lib/jmap/core/response/response_invocation.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+
+
+class ResponseInvocation {
```

Comment:
Why do we need a special type for responses? The spec do not make such distinction...

Reply:
It will help us to easier to deserialize response. anw, I will try to refactor it :)

Reply:
No I think that's not strictly necessary, we can live with minor imperfactions.